source/administration/plugins.rst

```diff
@@ -20,6 +20,20 @@ The Plugin Marketplace is a collection of plugins that can greatly increase the
 
 .. image:: https://user-images.githubusercontent.com/915956/66891467-1b18eb80-ef9e-11e9-9de3-37a3c5899bd8.png
 
+Plugin Labels
+~~~~~~~~~~~~~~~~
+
+Plugins in the Marketplace are labeled to make it easier for administrators to choose plugins that fit their company's security and risk policies if they do not allow for community plugins to be used. 
+
+**Mattermost Official Plugins**
+Plugins identified as "Official" are owned and managed by the Mattermost development team and are available in an open-source licencing repository for review.  These plugins are supported by our customer assistance team in production environments. 
+
+**Community Plugins**
```

Comment:
It's worth noting that there are no community nor beta plugins right now: should we defer the labelling documentation altogether until that changes?

Reply:
I just didn't want it to slip as a task and have the link leading to nothing.  I'm fine with pulling this back and publish later.